_pages/ato/levels.md

```diff
@@ -17,14 +17,15 @@ For more information, see [NIST 800-18](http://csrc.nist.gov/publications/nistpu
 
 Note that Open Data won't be listed, as it was developed within 18F.
 
-### Criteria for Open Data ATOs
+### Open Data ATOs
 
-The following are signs that your ATO process cannot qualify for as Open Data, and thus will be more complicated:
+To qualify for an Open Data ATO, all of the following must be true:
 
-* Are you dealing with Sensitive [Personally Identifiable Information (PII)](../../security/pii/)?
-* Any authentication?
-* Any authorization?
-* Is this the only place in the world you can do x? Is that x fundamental to what that agency does? (e.g. EPA tracking hazardous waste)
-* Is the data not open?
+* You are not dealing with Sensitive [Personally Identifiable Information (PII)](../../security/pii/)
+* There is no user authentication involved
+* There is no user authorization involved
+* This is not the only place in the world to complete a particular task
+* This functionality is not fundamental to what an agency does (e.g. the EPA tracking hazardous waste)
```

Comment:
Maybe I'm nitpicking, but I'm not sure 
```Is this the only place in the world you can do x? Is that x fundamental to what that agency does? (e.g. EPA tracking hazardous waste)```
is equivalent to 
```This is not the only place in the world to complete a particular task```
```This functionality is not fundamental to what an agency does (e.g. the EPA tracking hazardous waste)```

I'd interpret the top as: Is this the only place in the world you can do x? _If so_, is that x fundamental...
We can do unique/new things for an agency as long as that functionality isn't core to the agency's business.

If an Open Data ATO maps to a Confidentiality/Integrity/Availability of none/low/low, then building a new thing that the agency (or the agency's customers) depend on would require a higher availability rating (at least).

If it is a unique new thing that is tangential to the agency's business, it could still be marked as low (having a limited adverse effect), possibly depending on _how_ tangential it was.

Of course, it all comes down to @NoahKunin 's intention with that. :)